backend/tests/graphql/test_onboarding_request.py

```diff
@@ -0,0 +1,37 @@
+from app.graphql import schema as graphql_schema
+
+
+def test_create_onboarding_request():
+    query_string = """mutation testCreateOnboardingRequest {
+                        createOnboardingRequest(
+                            userInfo:
+                                {contactName: "Jane Doe",
+                                contactEmail: "janedoe@email.com",
+                                contactPhone: "12345",
```

Comment:
Does this test still work with the new validation? Also might be good to add a test for the validation logic itself.

Reply:
removed validation as discussed